packages/client/ui/react-ui/src/components/auth/PasskeyPrompt.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button, Dialog, Transition } from "@headlessui/react";
+import type React from "react";
+import { Fragment, type ReactNode } from "react";
+import type { UIConfig } from "@crossmint/common-sdk-base";
+
+import FingerprintIcon from "../../icons/fingerprint";
+import PasskeyIcon from "../../icons/passkey";
+import PasskeyPromptLogo from "../../icons/passkeyPromptLogo";
+import { PoweredByCrossmint } from "../common/PoweredByCrossmint";
+import { classNames } from "@/utils/classNames";
+
+const primaryButtonStyles = (props: { appearance?: UIConfig }): React.CSSProperties => ({
+    padding: "0.875rem",
```

Comment:
Forgot to mention in other PR, why not use Tailwind here?

Reply:
I updated to use a tailwind class 👍

Reply:
@AlbertoElias 
Actually, i'm running into a weird issue with the headless UI button background color not setting. 
It works by applying `important!` but that's not very good imo. 

I'm gonna revert this back to inline styles just in case
![Screenshot 2024-09-25 at 10 15 31 AM](https://github.com/user-attachments/assets/2ac3ec15-9f06-409b-948e-d3d018bea769)
![Screenshot 2024-09-25 at 10 16 15 AM](https://github.com/user-attachments/assets/a329cdb1-da14-45b7-8039-1abf0fffc9be)